castle/cms/tests/test_status.py

```diff
@@ -24,4 +24,4 @@ def test_status_control_panel_exists(self):
         if string in view:
             self.assertTrue(True, msg='None')
         else:
-            self.assertTrue(False, msg='cant find status control panel')
+            self.assertTrue(False, msg='can``t find status control panel')
```

Comment:
Ah, I meant that these have to be regular quote characters, not backquotes or backticks. Does this work?